cypress/integration/my-orcid/sidebar-add-websites-negative-testing.e2e.js

```diff
@@ -0,0 +1,506 @@
+  })
+
+  it('Internationalized (valid) URLs are not allowed', function () {
+    /* Business rule: valid internationalized URLs are NOT supported */
+    //click on edit pencil for websites section
+    cy.get('#websites-panel').within(($myPanel) => {
+      cy.get('#edit-button').click()
+    })
+    cy.get('#add-link').click()
+    cy.get('#description-input')
+      .clear()
+      .type(testingData.sidebarWebsitesURL.titleURL)
+    cy.get('#url-input')
+      .clear()
+      .type(testingData.sidebarWebsitesURL.internationalURL1)
+    cy.get('#save-websites-button').click()
+    //verify error message is displayed
+    cy.get('mat-error').should(
+      'have.text',
+      testingData.errorMessages.invalidURL
+    )
+
+    cy.get('#url-input')
+      .clear()
+      .type(testingData.sidebarWebsitesURL.internationalURL2)
+    cy.get('#save-websites-button').click()
+    //verify error message is displayed
+    cy.get('mat-error').should(
+      'have.text',
+      testingData.errorMessages.invalidURL
+    )
+
+    cy.get('#url-input')
+      .clear()
+      .type(testingData.sidebarWebsitesURL.internationalURL3)
+    cy.get('#save-websites-button').click()
+    //verify error message is displayed
+    cy.get('mat-error').should(
+      'have.text',
+      testingData.errorMessages.invalidURL
+    )
+
+    cy.get('#url-input')
+      .clear()
+      .type(testingData.sidebarWebsitesURL.internationalURL4)
+    cy.get('#save-websites-button').click()
+    //verify error message is displayed
+    cy.get('mat-error').should(
+      'have.text',
+      testingData.errorMessages.invalidURL
+    )
+
+    cy.get('#cancel-websites-button').click()
+  })
+
+  it('Error messages display according to the language selected', function () {
+    //switch to Spanish
+    cy.get('[aria-label="language"]').click()
+    cy.get('[id^=cdk-overlay]').within(($menuLanguage) => {
+      cy.get('button[role="menuitem"]')
+        .contains(testingData.errorMessages.displayLanguageSpanish)
+        .click()
+    })
+    cy.wait(3000) //wait for language to refresh
+    //click on edit pencil for websites section
+    cy.get('#websites-panel').within(($myPanel) => {
+      cy.get('#edit-button').click()
+    })
+    cy.get('#add-link').click()
+    cy.get('#description-input')
+      .clear()
+      .type(testingData.sidebarWebsitesURL.titleURL)
+    cy.get('#url-input').clear() //empty
+    cy.get('#save-websites-button').click()
+
+    //verify the URL field is required
+    cy.get('mat-error').should(
+      'have.text',
+      testingData.errorMessages.requiredURLSpanish
+    )
+
+    cy.get('#cancel-websites-button').click()
+
+    //switch back to English
+    cy.get('[aria-label="idioma"]').click()
+    cy.get('[id^=cdk-overlay]').within(($menuLanguage1) => {
+      cy.get('button[role="menuitem"]')
+        .contains(testingData.errorMessages.displayLanguageEnglish)
+        .click()
+    })
+    cy.wait(3000) //wait for language to refresh before Log out
```

Comment:
We can avoid using the wait 
```suggestion
   cy.get('[lang="en"]') //wait for language to refresh
```

Reply:
Done